en/advanced_config/static_pressure_buildup.md

```diff
@@ -0,0 +1,34 @@
+# Static Pressure Buildup
+
+Air flowing over an enclosed vehicle can cause the *static pressure* to change within the canopy/hull. 
+Depending on the location of holes/leaks in the hull, you can end up with under or overpressure (similar to a wing).
+
+The change in pressure can affect barometer measurements, leading to an inaccurate altitude estimate.
+This might manifest as the vehicle losing altitude during [Altitude Mode](../flight_modes/altitude_mc.md) (because the static pressure drops when it is no longer moving, the sensor reports a higher altitude, so the vehicle compensates by descending).
+
+One solution is to use foam-filled venting holes to reduce the buildup (as much as possible) and then attempt dynamic calibration to remove any remaining effects.
+
+> **Tip** Before "fixing" the problem you should first check that the Z setpoint tracks the estimated altitude (to verify that there are no controller issues).
+
+<span></span>
+> **Note** While it is possible to remove the barometer from the altitude estimate (i.e. only use altitude from the GPS), this is not recommended.
+  GPS is inaccurate in many environments, and particularly in urban environments where you have signal reflections off buildings.
+
+## Airflow Analysis
+
+You can modify the hull by drilling holes or filling them with foam.
+
+One way to analyse the effects of these changes is to mount the drone on a car and drive around with the hull exposed to air/wind.
```

Comment:
Interesting approach :+1: 
You might not want to do that on the slope of a hill though.

Reply:
Approach is from Lorenz - I think it is quite clever. I've modified slightly to say "on a relatively level surface).